pbqt.so/src/action.cpp

```diff
@@ -480,18 +480,36 @@ int pbqt_action::create_threads() {
         return -1;
       }
 
+      std::string srcpcibdf;
```

Comment:
keeping with other larger variable names, see if you can use snake case for these variable names?

Reply:
True, followed the convention already in place. Better we stick with it for now.